build/build-viewer.js

```diff
@@ -18,7 +18,7 @@ const glob = promisify(require('glob'));
 const lighthousePackage = require('../package.json');
 const makeDir = require('make-dir');
 const rimraf = require('rimraf');
-const uglifyEs = require('uglify-es'); // Use uglify-es to get ES6 support.
+const Terser = require('terser');
```

Comment:
nit:
```suggestion
const terser = require('terser');
```
unless there's a reason to capitalize?

Reply:
their documentation shows `Terser`. That's  the only justification prob
https://github.com/terser-js/terser#api-reference

uglifyjs does `UglifyJS`,  so terser probably copied  that

and uglfiyjs is a very old school js lib with quirks internally (imo)

Reply:
@brendankenny do I change capitalization? I copied it from their docs like @connorjclark mentioned.

Reply:
> @brendankenny do I change capitalization?

I personally would prefer it (we don't use it as a constructor or class), but if you feel differently and want to match the docs that works too.

Reply:
I have no real opinion on this 😄